src/lib/bootloader/grub2_widgets.rb

```diff
@@ -1055,66 +1064,82 @@ def loader_location_widget?
       end
 
       def generic_mbr_widget?
-        (Yast::Arch.x86_64 || Yast::Arch.i386) && grub2.name != "grub2-efi"
+        (Yast::Arch.x86_64 || Yast::Arch.i386) && !["grub2-efi", "grub2-bls"].include?(grub2.name)
       end
 
       def secure_boot_widget?
-        Systeminfo.secure_boot_available?(grub2.name)
+        Systeminfo.secure_boot_available?(grub2.name) && grub2.name != "grub2-bls"
       end
 
       def trusted_boot_widget?
-        Systeminfo.trusted_boot_available?(grub2.name)
+        Systeminfo.trusted_boot_available?(grub2.name) && grub2.name != "grub2-bls"
       end
 
       def update_nvram_widget?
-        Systeminfo.nvram_available?(grub2.name)
+        Systeminfo.nvram_available?(grub2.name) && grub2.name != "grub2-bls"
```

Comment:
in general I do not like to have it in widgets...this check for grub2.name should be done in given Systeminfo modules which do decision in single place regarding archs and bootloader types.